articles/azure/azs-how-add-data-managed-portal.md

```diff
@@ -0,0 +1,113 @@
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. In the *favourites* panel, select **Virtual machines**.
+
+    ![Virtual machines button](images/azsp_vmsmenu.png)
+
+3. In the *virtual machines* blade select the virtual machine you would like to add the disk to and then select **Disks** in the *settings* section.
+
+    ![Virtual machine disk button](images/azs-browser-button-vm-disks.png)
+
+4. In the *Disks* blade select **+Add data disk**.
+
+    ![Add data disk button](images/azs-browser-button-add-disk.png)
+
+# [Pre-existing disk](#tab/tabid-a)
+
+1. Enter the LUN (Logical unit number).
+
+    ![LUN input](images/azs-browser-input-disk-lun.png)
+
+2. Select the disk that you desire to add to the virtual machine.
+
+    ![Disk selection](images/azs-browser-input-disk-name.png)
+
+3. Select **Save**.
+
+    ![Save button](images/azs-browser-button-save-add-data-disk.png)
+
+4. You can monitor the progress of your deployment by clicking the **Notifications** icon.
+
+    ![Notification progress alert](images/azs-browser-notification-disks-progress.png)
+
+5. When the deployment is finished, the notification will change to *Successfully updated virtual machine disks*.
+
+    ![Notification complete alert](images/azs-browser-notification-disks-complete.png)
+
+6. You can view the disk by navigating to the Virtual machine that the disk was added to and then then selecting **Disks** from the *settings* section.
+
+    ![VM Data disks](images/azs-browser-disks.png)
+
+# [New disk](#tab/tabid-b)
+
+1. Enter the Logical unit number (LUN).
+
+    ![LUN input](images/azs-browser-input-disk-lun.png)
+
+2. In the name field select **Create disk** and enter the following information:
+
+    - Name - The name of the disk.
+
+    - Resource group - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - Location - This will be frn00006, which is the location of the Azure Stack.
+
+    - Account type - Standard disks (HDD) are backed by magnetic drives and are preferable for applications where data is accessed infrequently.
+
+        > [!Note]
+        > Currently only standard (HDD) disks are available in UKCloud for Azure Stack.
+
+    - Source type - Create a disk from a snapshot of another disk, a blob in a storage account, or create an empty disk.
+
+    - Size - The size of the disk in GB.
+
+3. You can monitor the progress of your deployment by clicking the **Notifications** icon.
+
+    ![Notification progress alert](images/azs-browser-notification-disks-progress.png)
+
+4. When the deployment is finished, the notification will change to *Successfully updated virtual machine disks*.
+
+    ![Notification complete alert](images/azs-browser-notification-disks-complete.png)
+
+5. You can view the disk by navigating to the virtual machine that the disk was added to and then selecting **Disks** from the *settings* section.
```

Comment:
Suggest:
You can view the disk by navigating to the VM that you added the disk to and then selecting **Disks**.